ansible/roles/monit/handlers/main.yml

```diff
@@ -0,0 +1,3 @@
+---
+  - name: monit
+    shell: monit stop all && monit reload && monit start all
```

Comment:
does `monit restart all` do the same thing? or does it not reload.  I think reload && restart all might work too and be simpler